discovery_engine_core/core/src/storage/sqlite.rs

```diff
@@ -489,7 +536,7 @@ impl SearchScope for SqliteStorage {
             .push(
                 "sd.documentId, nr.title, nr.snippet, nr.topic, nr.url, nr.image,
                 nr.datePublished, nr.source, nr.market, nc.domainRank, nc.score,
-                ur.userReaction, po.inBatchIndex, em.embedding
+                ur.userReaction, po.inBatchIndex, em.embedding, null as stackId
```

Comment:
here we never have a stack so no need to `JOIN` on the stack, still the mapping requires a `stackId` column to be included